packages/lib/services/commands/ToolbarButtonUtils.ts

```diff
@@ -52,8 +70,11 @@ export default class ToolbarButtonUtils {
 			tooltip: this.service.label(commandName),
 			iconName: command.declaration.iconName,
 			enabled: newEnabled,
-			onClick: () => {
+			onClick: async () => {
```

Comment:
This async was added by the the pre-commit hook

Reply:
I'm not sure it's actually the pre-commit hook, maybe it's when you rebased on dev? In any case this is fine. I've added a TypeScript so that all promises must either be awaited or explicitly marked as "void" (as the call to service.execute below).